### PR TITLE
[google.visualization] Correct CalendarOptions / Add Test

### PR DESCRIPTION
**Pull request by dckorben**
Please fill in this template.

- [ X ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ X ] Test the change in your own code. (Compile and run.)
- [ X ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ X ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ X ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ X ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ X ] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.google.com/chart/interactive/docs/gallery/calendar#configuration-options
- [ X ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

This adds a test and fixes incorrect nullability/missing properties on existing definition.

### CODE REVIEW

File: types/google.visualization/index.d.ts [hunk 1]
```diff
@@ -1368,32 +1372,33 @@ declare namespace google {
...
-                minValue: number;
-                values: number[];
+                colors?: string[];
+                maxValue?: number;
+                minValue?: number;
+                values?: number[];
             };
             forceIFrame?: boolean;
             height?: number;
             noDataPattern?: Object;
```
Comment (martin-badin) [inline types/google.visualization/index.d.ts:1396]: You can specify this one more

```suggestion
            noDataPattern?: {
            backgroundColor: string,
            color: string
            };
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/google.visualization/index.d.ts [hunk 2]
```diff
@@ -473,6 +473,10 @@ declare namespace google {
             strokeWidth?: number;
         }
 
+        export interface ChartStrokeOpacity extends ChartStroke {
```
Comment (martin-badin) [inline types/google.visualization/index.d.ts:476]: I am not sure but I think you can update `ChartStroke` to add `strokeOpacity`

Comment (martin-badin) [reply types/google.visualization/index.d.ts:476]: update: no, you can't

**Review by martin-badin — APPROVED** [review]

**Review by martin-badin — APPROVED** [review]

**Comment by dckorben** [comment]
Ready to merge

**Comment by dckorben** [comment]
Ready to merge

**Pull request merged**